pages/index.vue

```diff
@@ -17,10 +18,61 @@ export default {
 
     return { books, markers, themes: await themes }
   },
+  data () {
+    const body = marked(home.body)
+    return { body }
+  },
   mounted () {
     this.$store.commit('replaceFeatures', this.markers)
     this.$store.commit('enableInteraction')
-  },
-  components: { ThemeList }
+  }
 }
 </script>
+
+<style>
+@import "../components/colors/colors.css";
+
+.tagline {
+  position: absolute;
+  top: 5rem;
+  margin-top: 5vw;
+  width: 20vw;
+  min-width: 10rem;
+  max-width: 15rem;
+  padding: 1rem;
+}
+
+.tagline::before {
```

Comment:
This styling is a duplicate of theme-list. I think it is ok for now. but at some point, I do think we need to refactor the sidebar because the theme list overlaps the logo.

Reply:
I'd like to remove the theme list actually